decidim-core/app/packs/src/decidim/redesigned_heading_tag.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Ensures the aside heading tag is transformed to h2 or h1 if the drawer is shown or hidden
+ * @param {HTMLElement} node target node
```

Comment:
This is all in all a bad idea. I have ranted more about this at #9830 and I will continue my rants regarding this for the foreseeable future.

Reply:
I'll mark this as resolved as we are talking about the drawers #10184 and a possible removal will take place in a different PR.